Sources/Serialized.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Serialized.swift
+//  FTAPIKit-iOS
+//
+//  Created by Patrik Potoček on 27/06/2019.
+//  Copyright © 2019 FUNTASTY Digital s.r.o. All rights reserved.
+//
+
+import Foundation
+
+// This class provides user with easy way to serialize access to a property in multiplatform environment. This class is written with future PropertyWrapper feature of swift in mind.
```

Comment:
This should be public documentation.

```suggestion
/// This class provides user with easy way to serialize access to a property in multiplatform environment. This class is written with future PropertyWrapper feature of swift in mind.
```